### PR TITLE
Refactor: Change exchange / queue names

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

The idea of these changes is to change the names of the exchanges and the queues.

For the queues we decided to go with the following pattern `module_name.exchange_name.routing_key`.
For the exchanges we will go through it with @vk to categorize them correctly and merge the unneeded ones.

What component(s) does this PR affect?

- [x] Back-End (Database, Detection/Configuration/etc. Microservices)
- [x] Front-End (Flask, API, UI, etc)
- [x] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs (incl. wiki)
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs (incl. wiki)
- [ ] Build System

### Related Issue

Resolves #

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [x] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [x] This change requires a change in the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vkotronis** [comment]
Blocks https://github.com/FORTH-ICS-INSPIRE/artemis/pull/445

File: backend/entrypoint [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-make -j 4
+make -j
```
Comment (vkotronis) [inline backend/entrypoint:2]: why do we reduce the number of parallel jobs? Will this not delay the make process?

File: docker-compose.yaml [hunk 2]
```diff
@@ -188,6 +188,8 @@ services:
         expose:
             - ${RABBITMQ_PORT}
             # - 15672 management port
+        ports:
+            - 15672:15672
```
Comment (vkotronis) [inline docker-compose.yaml:192]: shouldn't those be set by the environment?

File: monitor/entrypoint [hunk 3]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-make -j 4
+make -j
```
Comment (vkotronis) [inline monitor/entrypoint:2]: same here (see previous comment)

File: testing/rpki/Dockerfile [hunk 4]
```diff
@@ -11,7 +11,7 @@ RUN apk update && apk add --no-cache postgresql-dev git libffi-dev cmake libssh
 RUN git clone -b v0.7.0 --depth 1 https://github.com/rtrlib/rtrlib.git
 WORKDIR /root/rtrlib
 RUN cmake -D CMAKE_BUILD_TYPE=Release .
-RUN make -j 4 && make install
+RUN make -j && make install
```
Comment (vkotronis) [inline testing/rpki/Dockerfile:14]: and here

**Review by vkotronis — COMMENTED** [review]
@slowr in general, this (the new format I mean) looks good to me. I did not find better names for the exchanges, could you point out specific examples that you would like to change?

Please also make sure the following:
1) ping me if you need any exchange names to change
2) see remaining (minor) comments
3) when ready, mark this as ready for review
4) update the PR so that it is in sync with the master branch

File: docker-compose.yaml (hunk 2 above)
Comment (slowr) [reply docker-compose.yaml:192]: oops, got through by mistake

File: backend/entrypoint (hunk 1 above)
Comment (slowr) [reply backend/entrypoint:2]: not specifying it means it uses all possible cores

**Comment by slowr** [comment]
> @slowr in general, this (the new format I mean) looks good to me. I did not find better names for the exchanges, could you point out specific examples that you would like to change?
> 
> Please also make sure the following:
> 
> 1. ping me if you need any exchange names to change
> 2. see remaining (minor) comments
> 3. when ready, mark this as ready for review
> 4. update the PR so that it is in sync with the master branch

How should we name the exchanges? any idea?

**Comment by vkotronis** [comment]
> > @slowr in general, this (the new format I mean) looks good to me. I did not find better names for the exchanges, could you point out specific examples that you would like to change?
> > Please also make sure the following:
> > 
> > 1. ping me if you need any exchange names to change
> > 2. see remaining (minor) comments
> > 3. when ready, mark this as ready for review
> > 4. update the PR so that it is in sync with the master branch
> 
> How should we name the exchanges? any idea?

You can leave them as-is for now, they are OK. Just make sure you fix the port mapping before merging.

**Review by vkotronis — APPROVED** [review]
Looks good to me as-is, you can remove the draft state if you also agree.

**Pull request merged**